AdvLoggerPkg/Library/AdvancedLoggerAccessLib/AdvancedLoggerAccessLib.c

```diff
@@ -136,6 +168,40 @@ FormatPhasePrefix (
   return (UINT16)PhaseStringLen;
 }
 
+/**
+  FormatDebugLevelPrefix
+
+  Adds a debug level indicator to the message being returned.  If debug level is recognized and specified,
+  returns the debug_level prefix in from the AdvMsgEntryPrefix, otherwise raw debug level value is returned.

```

Comment:
Putting the function name at the beginning of the function description is not necessary.

```suggestion
  Adds a debug level indicator to the message being returned.  If debug level is recognized and specified,
  returns the debug_level prefix in from the AdvMsgEntryPrefix, otherwise raw debug level value is returned.
```